libs/sdk-ui-web-components/src/visualizations/CustomElementAdapter.ts

```diff
@@ -0,0 +1,173 @@
+// (C) 2022 GoodData Corporation
+import React from "react";
+import ReactDom from "react-dom";
+import invariant from "ts-invariant";
+import { IAnalyticalBackend } from "@gooddata/sdk-backend-spi";
+import { LoadingComponent } from "@gooddata/sdk-ui/esm/base/react/LoadingComponent";
+
+import { CustomElementContext, getContext } from "../context";
+// eslint-disable-next-line import/no-unassigned-import
+import "./components.shadow.css";
```

Comment:
`*.shadow.css` has it's own loader that extracts the styles to a separate file, so we can load it with link tag. Regular `.css` files are simply loaded as `style` tag and auto-injected to `head`.